client/src/App.js

```diff
@@ -1,41 +1,36 @@
 import React, { useState, useEffect } from 'react';
 
-import Login from './components/LoginPage';
-import Register from './components/RegisterPage';
-import Home from './components/HomePage';
-import TempPlayer from './components/TempPlayer';
-import firebase  from './services/firebase';
+import { Login, Register, Home, TempPlayer } from './components';
```

Comment:
Is the new import statement working without any warnings or errors ? (line 3)

Reply:
It works well!

Reply:
Great!